### PR TITLE
build: bump rules_docker and instructions for installing bazelisk

### PR DESCRIPTION
**Pull request by subotic**

### CODE REVIEW

**Comment by subotic** [comment]
@SepidehAlassi Ok, now it should really work :-)

File: README.md [hunk 1]
```diff
@@ -75,10 +75,14 @@ export JAVA_HOME=`/usr/libexec/java_home -v 11`
 To install, follow these steps:
 
 ```
-$ brew install bazel
-$ brew upgrade bazel
+$ npm install -g @bazel/bazelisk
 ```
 
+This will install [bazelisk](https://github.com/bazelbuild/bazelisk) which is
```
Comment (SepidehAlassi) [inline README.md:81]: @subotic reading this documentation I expected `npm install -g @bazel/bazelisk` to also automatically install Bazel version 3.4.1, but that did not happen! 
Screenshot 2020-07-20 07 26 54

**Comment by SepidehAlassi** [comment]
It looks good with bazelisk, thanks for your work. 
However, there is a problem with documentation:

`# build webapi

$ bazel build //webapi`

`#run api (webapi)

$ bazel run //:api`

However, doing so one gets an error that the target 'api' is not declared in the package.
Screenshot 2020-07-20 08 33 07

I believe the correct target is webapi, so please either change the documentation to state
`#run api (webapi)

$ bazel run //:webapi`

Or change the configuration so that the target is `api`, then the documentation should be
`# build api

$ bazel build //api`

`#run api (api)

$ bazel run //:api`

**Comment by subotic** [comment]
@SepidehAlassi I tried to clarify the installation instructions a bit more and I've fixed the other thing. Can you please approve?

File: docs/05-internals/development/bazel.md [hunk 2]
```diff
@@ -4,21 +4,27 @@ The following section discusses on how to build and run tests for Knora-API
...
 ## Commands
 
 Build `webapi`:
 
 ```
 # build webapi
-$ bazel build //webapi
-
-# run api (webapi)
-$ bazel run //:api
```
Comment (SepidehAlassi) [inline docs/05-internals/development/bazel.md:18]: so there is no `bazel run` command to run the api anymore ? 
I remember there was a `bazel run \\...` to run everything, do you wanna add it to documentation?

Comment (subotic) [reply docs/05-internals/development/bazel.md:18]: There is. Just running the API does not make much sense without the whole stack. It won't work.

Comment (subotic) [reply docs/05-internals/development/bazel.md:18]: With `bazel` you can only run one target at a time. This is why we still have `make` and `docker-compose`.

**Comment by subotic** [comment]
@SepidehAlassi Thanks for the review :-)

**Pull request merged**